### PR TITLE
Fix behavior when encountering a bad embedding

### PR DESCRIPTION
**Pull request by JPPhoto**
Encountering a file in the embeddings directory without a `.pt` or `.bin` extension would cause InvokeAI to ask about reconfiguring models. This is because the embedding load error was never handled - it now is.

### CODE REVIEW

**Comment by JPPhoto** [comment]
To clarify, create an embedding with filesize zero that doesn't have a `.pt` or `.bin` extension (`touch embeddings/bob.pt:39`). This will throw an exception that's not caught at the right layer rather than giving a proper return value to indicate failure.

File: ldm/modules/textual_inversion_manager.py [hunk 1]
```diff
@@ -79,7 +79,10 @@ def load_textual_inversion(self, ckpt_path: Union[str,Path], defer_injecting_tok
 
         embedding_info = self._parse_embedding(str(ckpt_path))
 
-        if (
+        if embedding_info == None:
```
Comment (damian0815) [inline ldm/modules/textual_inversion_manager.py:82]: when comparing with `None`, use `is`, i.e. `if embedding_info is None`

**Review by damian0815 — APPROVED** [review]
looks good, the ==None thing is not a blocker

**Comment by JPPhoto** [comment]
Oh, python...

**Pull request merged**